main.tf

```
@@ -157,12 +156,14 @@ locals {
     "environment"      = local.task_container_environment
     "environmentFiles" = var.task_container_environment_file
     "MountPoints"      = local.task_container_mount_points
+    "ulimits"          = var.task_container_ulimits
     "logConfiguration" = {
       "logDriver" = "awslogs"
       "options"   = local.log_configuration_options
     }
     "privileged" : var.privileged
     "readonlyRootFilesystem" : var.readonlyRootFilesystem
+    "healthCheck" : var.container_health_check
   }, local.task_container_secrets, local.repository_credentials)
```

Following the pattern of some of the other optional variables (line 136-141), you could do something like

```suggestion
  }, local.task_container_secrets, local.repository_credentials, local.task_container_health_check)
  
  task_container_health_check  = length(var.container_health_check) > 0 ? { "healthCheck" = var.container_health_check } : null
```